src/client/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AuthenticationAgentActivity.cs

```diff
@@ -26,25 +25,25 @@ protected override void OnCreate(Bundle bundle)
             base.OnCreate(bundle);
             // Create your application here
 
-            WebView webView = new WebView(ApplicationContext);
```

Comment:
>ApplicationContext [](start = 42, length = 18)

Is it well understood what the difference is here? I assume this ill not have any draw backs as this is the actual activity hosting the webview, but was wondering what it even meant to use the ApplicationContext?